catalyst_voices/packages/catalyst_voices_shared/lib/src/common/build_config.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:catalyst_voices_shared/src/common/build_environment.dart';
+
+/// `BuildConfig` holds configuration values for the application.
+///
+/// To pass configuration values to the application,
+/// set the environment variables
+/// with `--dart define` when running the application:
+/// ```
+/// flutter run --dart-define SENTRY_DSN=https://dev.api.example.com --dart-define SENTRY_ENVIRONMENT=qa
+/// ```
+/// To access the environment variables, use the `BuildConfig` class.
+/// For example, to get the `SENTRY_DSN` environment variable,
+///  use `AppConfig.sentryDsn`.
+final class BuildConfig {
+  /// Gets the 'SENTRY_DSN' from the environment variables.
+  static String sentryDsn = const String.fromEnvironment(
```

Comment:
This field is neither const nor final, it can be changed from any place in the code. Was that on purpose?

Reply:
The purpose of the field is to pass the sentry dsn address during build time.

Reply:
Ok, then should I be allowed to reassign the field during the runtime? Currently it is be possible.

Reply:
Yes, and I don't know atm if we need it. What's your suggestion?

Reply:
Just declaring it as const. I don't see any valid reason to change it in runtime.

```dart
 static const String sentryDsn = String.fromEnvironment(
```